### PR TITLE
[RIL-1032] Allow 'bootable' query

### PR DESCRIPTION
**Pull request by aostapenko**
Allows to query volumes for non-admin users by bootable flag

### CODE REVIEW

File: cinder/files/liberty/cinder.conf.controller.Debian [hunk 1]
```diff
@@ -87,6 +87,8 @@ control_exchange = cinder
 notification_driver = messagingv2
 {%- endif %}
 
+# Available queries for non-admin user
+query_volume_filters_opt = name,status,metadata,availability_zone,bootable
```
Comment (fpytloun) [inline cinder/files/liberty/cinder.conf.controller.Debian]: I would prefer having this configurable with some default if not set.

Comment (aostapenko) [reply cinder/files/liberty/cinder.conf.controller.Debian]: default is "query_volume_filters_opt = name,status,metadata,availability_zone" in cinder source code

Comment (cznewt) [reply cinder/files/liberty/cinder.conf.controller.Debian]: this should definitely be configurable, create in map.jinja
'query_volume_filters': []
and then overridable thru model:
control:
  query_volume_filters:
  - name
  - status

make condifition if the query_volume_filters is not [] then print it

**Review by cznewt — CHANGES_REQUESTED** [review]
Make the changes to Mitaka, Newton, Ocata configs please

**Comment by aostapenko** [comment]
updated. Review please

**Pull request merged**